doc/lua_api.txt

```diff
@@ -2573,6 +2573,31 @@ Elements
     * All provided states must be active for the style to apply.
 * See [Styling Formspecs].
 
+### `set_focus[<name>;<always set>]`
+
+* Sets the focus to the element with the same `name` parameter.
+* **Note**: This element must be placed before the element it focuses.
+* `always set` (optional, default `false`): If false, this sets the focus to the
+  element specified only if the formspec is not being reshown. For instance, if
+  the formspec with the name "test:example" is shown, the focus will be set, but
+  if the same formspec is shown again consecutively, the focus will be set to
+  the last thing the user focused before reshowing. If true, the focus will always
+  be set in any circumstance.
```

Comment:
This is kinda long. How about:
```
* `force` (optional): By default (`false`), focus is not applied for
  re-sent formspecs with the same name so that player-set focus is kept.
  `true` sets focus to a the specified element for each sent formspec.
```